modules/customizing-the-web-console-URL.adoc

```diff
@@ -3,67 +3,60 @@
 // * web_console/customizing-the-web-console.adoc
 
 [id="customizing-the-web-console-url_{context}"]
-= Customizing the web console URL
+= Customizing console routes
 
-You can update the web console URL, `consoleURL`, to a custom value.
+For `console` and `downloads` routes, custom routes functionality is now implemented to use the new `ingress` config route configuration API. The Console Operator already contained custom route customization, but for the `console` route only. Now, if the `console` custom route is set up in both the `ingress` config and `console-operator` config, then the new `ingress` config custom route configuration takes precedent, because the route configuration via `console-operator` config is being deprecated.
+
+[id="customizing-the-console-route_{context}"]
+== Customizing the console route
+
+.Prerequisites
+
+* Log in to the cluster as a user with administrative privileges.
 
 .Procedure
 
-. Modify the cluster instance created by default during installation in the `consoles.operator.openshift.io` custom resource:
-+
-[source,terminal]
-----
-$ oc patch consoles.operator.openshift.io cluster --patch '{"spec":{"route":{"hostname":"console.example.com"}}}' --type=merge
-----
+* Set the custom hostname and, if needed, set the TLS on the ingress config's spec:
 +
 [source,yaml]
 ----
-apiVersion: operator.openshift.io/v1
-kind: Console
+apiVersion: config.openshift.io/v1
+kind: Ingress
 metadata:
   name: cluster
 spec:
-  route:
-    hostname: 'console.example.com'
+  componentRoutes:
+    - name: console
+      namespace: openshift-console
+      hostname: <custom-hostname>
+       secret:
+         name: <secret-name>
 ----
-. If you specify a custom certificate, you must create a secret in the `openshift-config` namespace that has the key and certificate. For example:
 +
-[source,terminal]
-----
-$ oc create secret tls console-tls --key=key.pem --cert=cert.pem -n openshift-config
-----
+If the custom hostname suffix does not match the cluster domain suffix, then a secret that contains a TLS certificate and key must exist in the `openshift-config` namespace and must be referenced in the `ingress` config. If the custom hostname suffix matches the cluster domain suffix, then the secret is optional. The secret must contain `tls.crt` and `tls.key` data keys, where the certificate and key are stored, and both must be in a valid format.
```

Comment:
A content comment--I think we want to keep the word "domain" in here, like maybe "If the domain for the custom hostname does not match the cluster domain,..."